OpenMEEG/include/triangle.h

```diff
@@ -50,7 +50,12 @@ namespace OpenMEEG {
 
         /// Constructors
 
+        #ifdef _MSC_VER
+        #pragma warning( suppress : 4245 )  // 'initializing': conversion from 'int' to 'unsigned int', signed/unsigned mismatch
```

Comment:
@papadop I feel like the solution here must be easy but I did not manage to figure it out in a few tries